### PR TITLE
[morpho-delegation] morpho-delegation strategy

### PR DESCRIPTION
**Pull request by Jean-Grimal**
Adds the morpho-delegation strategy that enables any token that inherits from Morpho Labs `DelegationToken` contract to get holders delegated voting power.

### CODE REVIEW

File: src/strategies/morpho-delegation/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, delegatedVotingPower]) => [
+      address,
+      parseFloat(formatUnits(delegatedVotingPower, options.decimals))
+    ])
+  );
+}
```
Comment (ChaituVR) [inline src/strategies/morpho-delegation/index.ts:34]: Hi @Jean-Grimal You can use the `contract-call` strategy in your space, we don't need to create a new strategy, you tried it on your space?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/morpho-delegation/index.ts (hunk 1 above)
Comment (Jean-Grimal) [reply src/strategies/morpho-delegation/index.ts:34]: Hi
Mb I wasn't aware of the `contract-call` strategy. It seems to work for us.

Comment (Jean-Grimal) [reply src/strategies/morpho-delegation/index.ts:34]: I tested it works

File: src/strategies/morpho-delegation/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# morpho-delegation
+
+This strategy allows to get any address' voting power for tokens inheriting from Morpho Labs Delegation Token.
```
Comment (julien-devatom) [inline src/strategies/morpho-delegation/README.md:3]:
```suggestion
This strategy allows to get any address' voting power for tokens inheriting from Morpho Labs Delegation Token.
It sums the delegated assets plus the balance of the user (if not delegated)
```

**Review by julien-devatom — APPROVED** [review]

**Comment by Jean-Grimal** [comment]
Hey @ChaituVR, we finally reoppened the PR as we decided to change the vote accountings, and the `contract-call` strategy didn't work anymore

File: src/strategies/morpho-delegation/index.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import { ADDRESS_ZERO } from '@uniswap/v3-sdk';
+
+export const author = 'Jean-Grimal';
```
Comment (ChaituVR) [inline src/strategies/morpho-delegation/index.ts:5]:
```suggestion
export const author = 'morpho-labs;
```

File: src/strategies/morpho-delegation/index.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+export const version = '0.1.1';
```
Comment (ChaituVR) [inline src/strategies/morpho-delegation/index.ts:6]:
```suggestion
export const version = '0.1.0';
```

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
@Jean-Grimal I will merge once you add the above changes but I am ok to proceed without them as well

**Pull request merged**